src/index.html

```diff
@@ -1,16 +1,52 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+   >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      data-qa="big"
+      class="searchForm"
+    >
+      <img
+        src="/images/Search.svg"
+        alt="searchIcon"
+        class="iconBig"
+      >
+      <input
+        type="text"
+        class="inputBig inputBorder"
```

Comment:
[checklist p.2 ](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

Reply:
ok, I misunderstood this part
I`ll fix it